EIPS/eip-6800.md

```diff
@@ -139,13 +139,17 @@ def old_style_address_to_address32(address: Address) -> Address32:
 These are the positions in the tree at which block header fields of an account are stored.
 
 ```
+def hash_point_to_bytes(point: Point) -> int:
+    return group_to_scalar_field(point).to_bytes(32, 'little')
```

Comment:
For more info: https://github.com/gballet/go-ethereum/pull/401 & https://github.com/ethereum/go-verkle/pull/428

Reply:
Note that `group_to_scalar_field` is defined on L38.